ORBIT/core/defaults/processes.yaml

```diff
@@ -0,0 +1,82 @@
+cure_grout:
```

Comment:
This is the new organization system for all things process related. This would be the default values for these processes. Note how some processes don't have constraints and some do. Some have other related variables as well (ie. mono_embed_len). `duration: dynamic` indicates that there is a dynamic calculation in the code (however you can pass in a fixed # instead to route around the dynamic calculation).